File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciordems/d/OrCommonFieldsDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.iih.ci.ord.ciordems.d
{
    /// <summary>
    /// 医疗安中公共字段
    /// </summary>
   public class OrCommonFieldsDTO : BaseDTO
    {
        /// <summary>
        /// 总次数
        /// </summary>
        public int? Times_cur
        {
            get { return getAttrVal<int?>("Times_cur", null); }
            set { setAttrVal<int?>("Times_cur", value); }
        }
        /// <summary>
        /// 医嘱天数
        /// </summary>
        public int? Use_days
        {
            get { return getAttrVal<int?>("Use_days", null); }
            set { setAttrVal<int?>("Use_days", value); }
        }
        /// <summary>
        /// 开始日期
        /// </summary>
        public DateTime? Dt_begin_ui
        {
            get { return getAttrVal<FDateTime>("Dt_begin_ui", null); }
            set { setAttrVal<FDateTime>("Dt_begin_ui", value); }
        }

        /// <summary>
        /// 结束日期
        /// </summary>
        public DateTime? Dt_end_ui
        {
            get { return getAttrVal<FDateTime>("Dt_end_ui", null); }
            set { setAttrVal<FDateTime>("Dt_end_ui", value); }
        }
        /// <summary>
        /// 在院执行次数
        /// </summary>
        public int? Times_mp_in
        {
            get { return getAttrVal<int?>("Times_mp_in", null); }
            set { setAttrVal<int?>("Times_mp_in", value); }
        }
        /// <summary>
        /// 标准价
        /// </summary>
        public FDouble Pri_std
        {
            get { return getAttrVal<FDouble>("Pri_std", null); }
            set { setAttrVal<FDouble>("Pri_std", value); }
        }

        /// <summary>
        /// 价格系数
        /// </summary>
        public FDouble Pri_ratio
        {
            get { return getAttrVal<FDouble>("Pri_ratio", null); }
            set { setAttrVal<FDouble>("Pri_ratio", value); }
        }

        /// <summary>
        /// 患者价格分类
        /// </summary>
        public string Id_pripat
        {
            get { return getAttrVal<string>("Id_pripat", null); }
            set { setAttrVal<string>("Id_pripat", value); }
        }
       /// <summary>
       /// 申请单号
       /// </summary>
        public string Applyno
        {
            get { return getAttrVal<string>("Applyno", null); }
            set { setAttrVal<string>("Applyno", value); }
        }
       /// <summary>
       /// 医疗单URL
       /// </summary>
        public string Funcclassstr
        {
            get { return getAttrVal<string>("Funcclassstr", null); }
            set { setAttrVal<string>("Funcclassstr", value); }
        }
       /// <summary>
       /// 医疗单id
       /// </summary>
        public string Id_srvof
        {
            get { return getAttrVal<string>("Id_srvof", null); }
            set { setAttrVal<string>("Id_srvof", value); }
        }
       /// <summary>
       /// 医嘱来源方式类型
       /// </summary>
        public string Eu_orsrcmdtp
        {
            get { return getAttrVal<string>("Eu_orsrcmdtp", null); }
            set { setAttrVal<string>("Eu_orsrcmdtp", value); }
        }
       /// <summary>
       /// 简化的流程标识
       /// </summary>
        public bool? Fg_quickwflow
        {
            get { return getAttrVal<bool?>("Fg_quickwflow", null); }
            set { setAttrVal<bool?>("Fg_quickwflow", value); }
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciordems/d/OrCommonFieldsDTO.java
package iih.ci.ord.ciordems.d;

import xap.mw.core.data.BaseDTO;
import xap.mw.coreitf.d.FBoolean;
import xap.mw.coreitf.d.FDateTime;
import xap.mw.coreitf.d.FDouble;

public class OrCommonFieldsDTO extends BaseDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	/**
	 * 总次数
	 * @return Integer
	 */
	public Integer getTimes_cur() {
		return ((Integer) getAttrVal("Times_cur"));
	}
	/**
	 * 总次数
	 * @param Times_cur
	 */
	public void setTimes_cur(Integer Times_cur) {
		setAttrVal("Times_cur", Times_cur);
	}
	
	/**
	 * 使用天数
	 * @return Integer
	 */
	public Integer getUse_days() {
		return ((Integer) getAttrVal("Use_days"));
	}
	/**
	 * 使用天数
	 * @param Use_days
	 */
	public void setUse_days(Integer Use_days) {
		setAttrVal("Use_days", Use_days);
	}
	
	/**
	 * 开始时间
	 * @return Integer
	 */
	public FDateTime getDt_begin_ui() {
		return ((FDateTime) getAttrVal("Dt_begin_ui"));
	}
	/**
	 * 开始时间
	 * @param Dt_begin_ui
	 */
	public void setDt_begin_ui(FDateTime Dt_begin_ui) {
		setAttrVal("Dt_begin_ui", Dt_begin_ui);
	}
	
	/**
	 * 结束时间
	 * @return Integer
	 */
	public FDateTime getDt_end_ui() {
		return ((FDateTime) getAttrVal("Dt_end_ui"));
	}
	/**
	 * 结束时间
	 * @param Dt_end_ui
	 */
	public void setDt_end_ui(FDateTime Dt_end_ui) {
		setAttrVal("Dt_end_ui", Dt_end_ui);
	}
	
	/**
	 * 在院执行次数
	 * @return Integer
	 */
	public Integer getTimes_mp_in() {
		return ((Integer) getAttrVal("Times_mp_in"));
	}
	/**
	 * 在院执行次数
	 * @param Times_mp_in
	 */
	public void setTimes_mp_in(Integer Times_mp_in) {
		setAttrVal("Times_mp_in", Times_mp_in);
	}
	
	/**
	 * 标准价
	 * @return FDouble
	 */
	public FDouble getPri_std() {
		return ((FDouble) getAttrVal("Pri_std"));
	}	
	/**
	 * 标准价
	 * @param Pri_std
	 */
	public void setPri_std(FDouble Pri_std) {
		setAttrVal("Pri_std", Pri_std);
	}
	/**
	 * 价格系数
	 * @return FDouble
	 */
	public FDouble getPri_ratio() {
		return ((FDouble) getAttrVal("Pri_ratio"));
	}	
	/**
	 * 价格系数
	 * @param Pri_ratio
	 */
	public void setPri_ratio(FDouble Pri_ratio) {
		setAttrVal("Pri_ratio", Pri_ratio);
	}
	/**
	 * 患者价格分类
	 * @return String
	 */
	public String getId_pripat() {
		return ((String) getAttrVal("Id_pripat"));
	}	
	/**
	 * 患者价格分类
	 * @param Id_pripat
	 */
	public void setId_pripat(String Id_pripat) {
		setAttrVal("Id_pripat", Id_pripat);
	}
	/**
	 * 申请单号
	 * @return
	 */
	public String getApplyno() {
		return ((String) getAttrVal("Applyno"));
	}	
	/**
	 * 申请单号
	 * @param Applyno
	 */
	public void setApplyno(String Applyno) {
		setAttrVal("Applyno", Applyno);
	}
	/**
	 * 医疗单URL
	 * @return
	 */
	public String getFuncclassstr() {
		return ((String) getAttrVal("Funcclassstr"));
	}	
	/**
	 * 医疗单URL
	 * @param Funcclassstr
	 */
	public void setFuncclassstr(String Funcclassstr) {
		setAttrVal("Funcclassstr", Funcclassstr);
	}
	/**
	 * 医疗单id
	 * @return
	 */
	public String getId_srvof() {
		return ((String) getAttrVal("Id_srvof"));
	}	
	/**
	 * 医疗单id
	 * @param Id_srvof
	 */
	public void setId_srvof(String Id_srvof) {
		setAttrVal("Id_srvof", Id_srvof);
	}
	/**
	 * 医嘱来源方式类型
	 * @return
	 */
	public String getEu_orsrcmdtp() {
		return ((String) getAttrVal("Eu_orsrcmdtp"));
	}	
	/**
	 * 医嘱来源方式类型
	 * @param Eu_orsrcmdtp
	 */
	public void setEu_orsrcmdtp(String Eu_orsrcmdtp) {
		setAttrVal("Eu_orsrcmdtp", Eu_orsrcmdtp);
	}
	/**
	 * 简化的流程标识
	 * @return
	 */
	public FBoolean getFg_quickwflow() {
		return ((FBoolean) getAttrVal("Fg_quickwflow"));
	}	
	/**
	 * 简化的流程标识
	 * @param Fg_quickwflow
	 */
	public void setFg_quickwflow(FBoolean Fg_quickwflow) {
		setAttrVal("Fg_quickwflow", Fg_quickwflow);
	}
}
